mlflow/statsmodels.py

```diff
@@ -464,7 +464,9 @@ def results_to_dict(results):
                         renamed_keys_dict = prepend_to_keys(d, f)
                         results_dict.update(renamed_keys_dict)
 
-                    elif isinstance(field, (int, float)):
+                    # `not isinstance(field, bool)` is required because `bool` is a subclass of
+                    # `int`
+                    elif not isinstance(field, bool) and isinstance(field, (int, float)):
```

Comment:
Should we convert `bool` to 0 or 1 instead of excluding it?

Reply:
cc @olbapjose

Reply:
I think it's acceptable to omit non-numeric values, including this boolean.

Can we define an ``is_numeric`` utility method somewhere in the `mlflow.utils` package that handles these cases? In addition to the boolean check, I think we can expand beyond `isinstance(field, (int, float))` to `isinstance(numbers.Number)`. We do this elsewhere in MLflow (e.g., https://github.com/mlflow/mlflow/blob/f2c854ec18a157c6bf1668f9432a20810da0ca2d/mlflow/utils/validation.py#L74)

Reply:
Sounds good. I'll add `is_numeric`

Reply:
Done!

Reply:
Thank you guys for fixing this, sorry for being late to the party :-)